### PR TITLE
[Review Needed] Fixing of CSS and JS and adding internal links

### LINKED ISSUE #638: Fix CSS and avoid external CSS of other services
The website had included external CSS. We should not share data with other services pro-actively. I deleted the services. We need help now to re-connect the original CSS with the current website.

### LINKED ISSUE #639: Deletion of external services e.g. jquery links broke site
Goal: Please fix the broken parts and only include links and Javascript etc. internally.

### PR DESCRIPTION
**Pull request by Sangatdas**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://sangatdas.github.io/gci18.fossasia.org/)

- #639
- #638  

### 1. Addition of jquery- 
Internal jquery has been added as many libraries depended on it. However addition of internal jquery induced some new issues such as broken slider and navbar issues.

### 2. Navbar Fix- 
Fixed navbar according to theme.

### 3. CSS Fix-
Fixed broken css and fontawesome

### 4. Image Slider Fix-
Fixed broken image slider using bxslider library instead of flexslider which confilcted with code.

### Before:
screenshot from 2018-12-23 16-12-22

screenshot from 2018-12-23 17-01-42

screenshot from 2018-12-23 16-12-38

screenshot from 2018-12-23 16-12-44

screenshot from 2018-12-23 16-12-47

screenshot from 2018-12-23 16-12-51

### After:
screenshot from 2018-12-25 18-39-06

screenshot from 2018-12-23 16-13-33

screenshot from 2018-12-23 16-13-50

screenshot from 2018-12-23 16-13-55

screenshot from 2018-12-23 16-13-58

screenshot from 2018-12-23 16-14-03

- - - - - - - - - - - -

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -97,17 +97,17 @@ <h6 class="title">Latest Updates</h6>
           </li>
           <li>
             <a href="https://gitter.im/fossasia/fossasia" target="_blank" rel="noopener noreferrer">
-              <i class="fab fa-gitter"></i>
+              <i class="fa fa-comment"></i>
```
Comment (NishithP2004) [inline _includes/footer.html]: Why did you remove this ?

File: _includes/footer.html [hunk 2]
```diff
@@ -97,17 +97,17 @@ <h6 class="title">Latest Updates</h6>
...
             </a>
           </li>
           <li>
             <a href="https://www.weibo.com/u/5784920339" target="_blank" rel="noopener noreferrer">
-              <i class="fab fa-weibo"></i>
+              <i class="fa fa-weibo"></i>
```
Comment (NishithP2004) [inline _includes/footer.html]: Why is this changed?

File: _includes/footer.html [hunk 3]
```diff
@@ -97,17 +97,17 @@ <h6 class="title">Latest Updates</h6>
...
             </a>
           </li>
           <li>
             <a href="https://vk.com/fossasia" target="_blank" rel="noopener noreferrer">
-              <i class="fab fa-vk"></i>
+              <i class="fa fa-vk"></i>
```
Comment (NishithP2004) [inline _includes/footer.html]: Why is this altered?

File: _includes/footer.html (hunk 1 above)
Comment (NishithP2004) [reply _includes/footer.html]: Why has Gitter been removed?

Comment (Sangatdas) [reply _includes/footer.html]: I have used fontawesome4.7 and it didn't have gitter icon. The link is working only the icon has been changed. I did this because fontawesome5 was giving garbage icons when used internally.

File: _includes/footer.html (hunk 2 above)
Comment (Sangatdas) [reply _includes/footer.html]: Again, I have used fontawesome 4.7 instead of fontawesome 5 since the later was not giving the expected icons

File: _includes/footer.html (hunk 3 above)
Comment (Sangatdas) [reply _includes/footer.html]: Fontawesome 4.7 uses fa instead of fab

File: _includes/footer.html (hunk 1 above)
Comment (NishithP2004) [reply _includes/footer.html]: But Gitter is required

Comment (NishithP2004) [reply _includes/footer.html]: Please use frontawesome 5 only.

File: _includes/footer.html (hunk 2 above)
Comment (NishithP2004) [reply _includes/footer.html]: Please use frontawesome 5 only

File: _includes/footer.html (hunk 3 above)
Comment (NishithP2004) [reply _includes/footer.html]: Please use frontawesome 5 only.

**Comment by NishithP2004** [comment]
Requesting a review @mariobehling @jogendra @nikit19

File: _includes/footer.html (hunk 1 above)
Comment (pranav1698) [reply _includes/footer.html]: The link to gitter should have the gitter icon

File: _includes/footer.html (hunk 2 above)
Comment (pranav1698) [reply _includes/footer.html]: The icon is correct and works for weibo, hence these changes are justified 
Also font awesome 5 icon for weibo is not working

**Comment by Sangatdas** [comment]
screenshot from 2018-12-28 01-30-12
screenshot from 2018-12-28 01-30-23

@NishithP2004 @pranav1698 Have added fontawesome 5 and removed fontawesome 4.7. Refer screenshots.

**Comment by NishithP2004** [comment]
What has happened to the `Thank you for contributing ` which should have a nice js effect.

**Comment by NishithP2004** [comment]
Just a suggestion, is there any way you can make the icons etc red? @Sangatdas

**Comment by NishithP2004** [comment]
Probably by changing the colour gradient.

**Comment by Sangatdas** [comment]
> What has happened to the `Thank you for contributing ` which should have a nice js effect.

Can you provide me with some video or something? I hadn't seen the site before this bug was reported.

**Comment by pranav1698** [comment]
@sangatdas you can check the history of commits using 'git log'and then
checkout any previous commit that would point to the previous site

**Comment by NishithP2004** [comment]
https://nishithp2004.github.io/gci18.fossasia.org you may refer this.

**Comment by KSNINJA** [comment]
You have missed custom.css . It can be added in this PR  .

**Comment by NishithP2004** [comment]
@Sangatdas Please do as requested for the PR to be merged.

**Comment by Sangatdas** [comment]
I hadn't seen the previous site earlier before making the PR. Now, that I've seen it, I'll be able to make the required changes

**Comment by Sangatdas** [comment]
I'll be closing this PR and request a new PR since I have to do a lot of changes. Thanks for the reviews and guidance
